api/test/validate/pgv_test.cc

```diff
@@ -19,7 +19,6 @@
 #include "envoy/config/filter/http/ip_tagging/v2/ip_tagging.pb.validate.h"
 #include "envoy/config/filter/http/lua/v2/lua.pb.validate.h"
 #include "envoy/config/filter/http/router/v2/router.pb.validate.h"
-#include "envoy/config/filter/http/squash/v2/squash.pb.validate.h"
```

Comment:
...and this to make it work

Reply:
oops yeah, I totally missed this one :-P